pkg/nimble/Makefile.dep

```diff
@@ -3,7 +3,9 @@ USEMODULE += sema
 USEMODULE += event_callback
 USEMODULE += ztimer_msec
 
-USEPKG += nrfx
+ifneq (,$(filter nrf51 nrf52 nrf53,$(CPU_FAM)))
+  USEPKG += nrfx
+endif
 
```

Comment:
Maybe for consistency 
```
ifneq (,$(filter esp32,$(CPU_FAM)))
  USEMODULE += esp_ble_nimble
endif
```

Reply:
Hm, IMHO the `nimble` package shouldn't know too much about dependencies for certain MCUs. I would rather move the dependency on the `nrfx` package to `cpu/nrf_common/Makefile.dep` than to add further dependencies to `nimble`'s `Makefile.dep`.

Reply:
Either way is fine, I just want some consistency.

Reply:
I moved the dependency on `nrfx` package to `cpu/nrf_common/Makefile.dep`.